cranelift/codegen/src/isa/x64/lower.isle

```diff
@@ -1760,10 +1758,8 @@
       ;; other sites did not permit sinking, then we would be missing the load
       ;; for other cmp-sites. TODO:
       ;; https://github.com/bytecodealliance/wasmtime/issues/3953.
```

Comment:
I think the comments here can be removed.

Reply:
Fixed!